operator/controllers/common/phasehandler.go

```diff
@@ -103,7 +100,7 @@ func (r PhaseHandler) handleCompletedPhase(state apicommon.KeptnState, piWrapper
 	if err := r.SpanHandler.UnbindSpan(reconcileObject, phase.ShortName); err != nil {
 		r.Log.Error(err, controllererrors.ErrCouldNotUnbindSpan, reconcileObject.GetName())
 	}
-	r.EventSender.SendK8sEvent(phase, "Normal", reconcileObject, "Succeeded", "has succeeded", piWrapper.GetVersion())
+	r.EventSender.SendK8sEvent(phase, "Normal", reconcileObject, apicommon.PhaseStateFinished, "has finished", piWrapper.GetVersion())
```

Comment:
sometimes we have has finished some time is finished, should we use a const for these messages too ?

Reply:
Sure, nice catch